tcf_website/templates/course/user_courses.html

```diff
@@ -0,0 +1,128 @@
+                  <strong>{{ saved.course }} </strong> — {{ saved.instructor.full_name }}
+              </a>
+              <p id="title" hidden> {{saved.course.subdepartment.mnemonic}} {{saved.course.number}} </p>
+              <button class="ml-auto btn btn-sm bg-tcf-indigo text-white unsave-btn" id="unsaveCourseBtn{{saved.id}}" type="button">
+                  <i class="fa fa-trash-o" aria-hidden="true"></i>
+                  Unsave
+              </button>
+            </div>
+            <div class="card-body">
+                <ul class="row mb-0 px-3">
+                    <li class="col text-nowrap">
+                        <small class="text-uppercase">
+                            <i class="fa fa-star"></i> Rating
+                        </small>
+                        <p class="h4" id="rating">
+                            {% if saved.average_rating %}
+                                {{ saved.average_rating|floatformat:2 }}
+                            {% else %}
+                                &mdash;
+                            {% endif %}
+                        </p>
+                    </li>
+                    <li class="col text-nowrap">
+                        <small class="text-uppercase">
+                            <i class="fa fa-dumbbell"></i> Difficulty
+                        </small>
+                        <p class="h4" id="difficulty">
+                            {% if saved.average_difficulty %}
+                                {{ saved.average_difficulty|floatformat:2 }}
+                            {% else %}
+                                &mdash;
+                            {% endif %}
+                        </p>
+                    </li>
+                    <li class="col text-nowrap">
+                        <small class="text-uppercase">
+                            <i class="fa fa-chart-bar"></i> GPA
+                        </small>
+                        <p class="h4" id="gpa">
+                            {% if saved.average_gpa %}
+                                {{ saved.average_gpa|floatformat:2 }}
+                            {% else %}
+                                &mdash;
+                            {% endif %}
+                        </p>
+                    </li>
+                    <div class="col-md-3">
+                        <small class="text-uppercase">
+                            <i class="fa fa-calendar"></i> Last Taught
+                        </small>
+                        <p class="h4">{{ saved.semester_last_taught }}</p>
+                    </div>
+                    <div class="col-sm col-md-4">
+                        <a href="#saveCourseModal{{saved.id}}" data-toggle="modal">
+                        <small class="text-uppercase">
+                            <i class="fa fa-sticky-note"></i> Notes
+                        </small>
+                        <p class="h4 notes-content" id="notes{{saved.id}}">{{ saved.notes }}</p>
```

Comment:
What size should the notes be? I think h4 might be too large since that shows only a few words, and people may want to write more detailed pro/con notes
I think it might look better in a smaller font and possibly with a larger/wider box

Reply:
I tried a smaller font, but it looked a little off given the other text in the row. I did make the box wider and improved the collapsing for longer notes, so it looks cleaner now :)